Automattic/crowdsignal-ui
# Update App styling structure

## Summary

The purpose of this PR is to refactor the `theme-compatibility` package to create a base stylesheet that will be loaded along with any theme and will be responsible for creating a basic/generic layout for the application, giving it a decent look N feel even without any theme loaded.

## Test Instructions

* Make sure that both, the `dashboard` and the `project-renderer` are running;
* Build the themes packages using `yarn workspace @crowdsignal/theme-compatibility build`;
* The application should work as before and the layout of the `editor` and the `project-renderer` should be quite the same.
